database/migrations/create_snomed_snap_concept_table.php.stub

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('snomed_snap_concept', function (Blueprint $table) {
```

Comment:
If you will make this package public, would look into having the table names configurable depending on whether someone wants to include a snapshot or the full version of the databse

Reply:
It is already configurable in the sense that when making use of the package you can publish the migration in your project and they will override the ones from the module.